operations/finance/operations-playbook.md

```diff
@@ -1,2 +1,242 @@
+|[Book self-serve and cloud billing orders](https://docs.google.com/document/d/1kGJz8lOIjhc5qeuSKcDcMcRiHB3YQ84aWYigP1wMXhA/edit?usp=sharing)|`Netsuite` `Stripe`|Playbook| |Daily|Jeff Dynda|Accounting|
+|[Cloud and sales-serve reconciliation](https://docs.google.com/document/d/1K6wjEx7LftW1t8og9EV3vaM9CRsQP7jmDDOfr4nHQ4Y/edit?usp=sharing)|`Netsuite` `Stripe`|Playbook| |Monthly|Jeff Dynda|Accounting|
+|[Recognize revenue for self-managed and cloud sales-serve and self-serve sales](https://docs.google.com/document/d/1PUK3qgsgQQ2NOLEEVZAvmeJgkCo4zXs99JKnnKappJk/edit)|`Netsuite`|Playbook| |Monthly|Ben Graham|Accounting|
+|[Accounts receivable reconciliation and rollforward](https://docs.google.com/document/d/1yYAIXQjCI17Zd3uRHj798WMNzkdpBImlR5Zs1vsimPw/edit?usp=sharing)|`Netsuite`|Playbook| |Monthly|Jeff Dynda|Accounting|
+|[Initiating customer credit freeze](https://docs.google.com/document/d/1TSCUqJh7SoWEpkeozshCDvnrT9sWglHDB5GUoGNXV6M/edit?usp=sharing)|`Netsuite` "SFDC`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Accounts receivable write-off](https://docs.google.com/document/d/1ogUGZ0jdVbJhHR7wYtB6_WBrCer4UWn7iX1r7cZIO_I/edit?usp=sharing)|`Netsuite`|Playbook| |Quarterly|Jeff Dynda|Accounting|
+|[Issuing SLA credits](https://docs.google.com/document/d/1YnvUA6jdAJF55kd4YaILpK_BfrM2MUnkM5XCUXa71MY/edit?usp=sharing)|`Netsuite` `Stripe`|Playbook| |As needed|Ben Graham|Accounting|
+|[Record deferred revenue](https://docs.google.com/document/d/1IzKya4OrMcFiiF4iuG9IrKsIcSEZ8YpOpdEPJNLHbnM/edit#heading=h.ocf5tbkimw96)|`Netsuite`|Playbook| |Monthly|Ben Graham|Accounting|
+|[Build revenue metrics for month-end close](https://docs.google.com/document/d/1pwIsEE4zSyIlENK9ALtTdIgK-LJem-gt_Ep-0rN0rsU/edit?usp=sharing)|`Netsuite`|Playbook| |Monthly|Ben Graham|Accounting|
+|[Create items and SKUs](https://docs.google.com/document/d/15rQQlI1hUE8fyKX6fiQ08ZNkiUKs8awSeP6ROYh2PDo/edit?usp=sharing)|`Netsuite`|Playbook| |As needed|Ben Graham|Accounting|
+
+
+
+### Prepaid assets
+Internal processes and framework for booking and recognizing upfront payments for future vendor services. 
+
+|Process Description/Doc Link|System/Tool|Document Type|Flow Chart|Occurrence|AOR|Access Rights|
+|:----|:----|:----|:----|:----|:----|:----|
+|[Prepaids assets month-end close](https://docs.google.com/document/d/1e1OTP2Ru1-vAEv_8PeqffY8JIuJlTzjcz0o1GpwIS1Q/edit)|`Netsuite`|Playbook| |Monthly|Ben Graham|Accounting|
+
+
+
+### Fixed assets
+Internal processes and framework for booking and recognizing phyiscal property and depreciation/amortization. 
+
+|Process Description/Doc Link|System/Tool|Document Type|Flow Chart|Occurrence|AOR|Access Rights|
+|:----|:----|:----|:----|:----|:----|:----|
+|[Fixed asset walkthrough](https://docs.google.com/document/d/1Weh3sAyNBvkO0mO_4IUGAltISTE96uEn4sYov9MbtD0/edit)|`Netsuite`|Process Narrative| |Monthly|Jeff Dynda|Accounting|
+|[Mattermost hardware buyback]|`Netsuite`|Playbook| |Quarterly|Jeff Dynda|Accounting|
```

Comment:
@TQuock same here, any specific link you want to include for hardware buyback?

Reply:
Thanks for highlighting, Jason! These are placeholders for now as we finalize the documentation. More PRs to come in the next few weeks.